### ISSUE TITLE
Feedback information

### ISSUE THREAD

**Issue by ElinaWihuri** [NatLibFi/Skosmos#1616]
## Description of the enhancement 
Could the email address be mandatory in the feedback?

## Who are the users that would benefit from the enhancement and how?
Users who need to be asked for more information in unclear feedback. + Updaters who can verify, if necessary, what is meant in the feedback. Maybe updaters also receive less spam.

(Name specific organizations, roles and/or communities if possible, and/or point to relevant discussion elsewhere)

## What new functionalities would the enhancement make possible?
Feedback processing may become easier. You can contact the feedback provider if necessary.

## Why is the enhancement important?
So that the feedback can be processed. It can be verified what issue or concept the feedback has concerned. 

(Why should this be prioritized, instead of spending energy on something else? Who would do the work to make this happen?)

**Comment by joelit** [comment]
Good call! The experience shows that valuable and actionable feedback is seldom anonymous, and vice versa. I'll add this to the development queue to be discussed.
